python/ray/tests/test_utils.py

```diff
@@ -108,6 +111,35 @@ def test_get_current_node_cpu_model_name():
         assert get_current_node_cpu_model_name() == "Intel Xeon"
 
 
+@pytest.mark.asyncio
```

Comment:
One last comment: could you move this to `test_list_actors.py`?

Reply:
tag @Catch-Bull